unix/vncserver/vncsession.c

```diff
@@ -507,7 +519,7 @@ static void
 usage(void)
 {
     fprintf(stderr, "Syntax:\n");
-    fprintf(stderr, "    vncsession [-D] <username> <display>\n");
+    fprintf(stderr, "    vncsession [-D] [-l path/to/log/dir] <username> <display>\n");
```

Comment:
Same thing here. Let's focus on one thing at a time.

Reply:
This was one of the things specifically requested by the original issue, which is why I included it here. I'm happy to move it to another PR, however, if you'd prefer that.

Reply:
Yes please.